Automattic/sensei
# Add course navigation block

Fixes #4369 

### Changes proposed in this Pull Request

* Add a course navigation block that renders the current course modules and lessons
* Add related styles & icons
* Improve collapsible script to handle collapsed items better
* Add block to course theme template
* Add basic styling & spacing to course theme template

### Testing instructions

* Enable `sensei_feature_flag_course_theme`
* Set a course to use the Sensei Course Theme
* Open it in the frontend and navigate lessons/modules
* Take course, complete lessons with a learner to check the lesson status updating


### Screenshot / Video

----
<img width="902" alt="image" src="https://user-images.githubusercontent.com/176949/142469269-61189411-9c9b-473e-9720-7b7ce85a1e03.png">
